cmd/common/utils/errors.go

```diff
@@ -35,6 +35,26 @@ func WrapInErrListNodes(err error) error {
 	return fmt.Errorf("failed to list nodes: %w", err)
 }
 
+// Manager
+
+func WrapInErrManagerInit(err error) error {
```

Comment:
I am not sure what it would look like. Could you please describe a little bit more in details your suggestion?

Reply:
I was thinking about something in terms of https://travix.io/errors-derived-from-constants-in-go-fda6748b4072, but I've realized it might be slightly overengineered. It would look something like this:
```go
func TestHelloWorld(t *testing.T) {
  if err := foo(); !errors.Is(err, &ManagerInitError{}) {
    t.Fatalf("Expected foo")
  }
}

func foo() error {
  return &ManagerInitError{
    Cause: fmt.Errorf("test"),
  }
}

type ManagerInitError struct {
  Cause error
}

func (m *ManagerInitError) Error() string {
  return fmt.Sprintf("initializing manager: %v", m.Cause)
}

func (m *ManagerInitError) Unwrap() error {
  return m.Cause
}

func (m *ManagerInitError) Is(target error) bool {
  _, ok := target.(*ManagerInitError)

  return ok
}
```

Reply:
When I tried I found the same so I thought I had misunderstood your suggestion 😅 .

Yeah, I agree that it would be too much for our use case.